WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Blaze/DashboardBlazeCampaignStatusView.swift

```diff
@@ -0,0 +1,86 @@
+import Foundation
+import UIKit
+import WordPressKit
+
+final class DashboardBlazeCampaignStatusView: UIView {
+    let titleLabel = UILabel()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        titleLabel.translatesAutoresizingMaskIntoConstraints = false
+        titleLabel.adjustsFontForContentSizeCategory = true
+        titleLabel.font = WPStyleGuide.fontForTextStyle(.caption2, fontWeight: .semibold)
+
+        addSubview(titleLabel)
+        pinSubviewToAllEdges(titleLabel, insets: .init(top: 4, left: 4, bottom: 3, right: 4))
```

Comment:
Wdyt of extracting the inset to a constant?

Reply:
Done.

I think it decreases readability, but if that's the established approach, I'll follow it 👍